CITATION.cff

```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+title: HADDOCK3
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
```

Comment:
This is wrong.

The citation should be something that matches a DOI, there is no publication so you need to use the one deposited in zenodo - https://zenodo.org/records/10527751

Then you can create a new `CITATION.cff` using `doi2cff`:

```bash
$ pip install git+https://github.com/citation-file-format/doi2cff
$ doi2cff init https://doi.org/10.5281/zenodo.10527751
$ catCITATION.cff
```

Reply:
ok nice